proto/protovalidate/buf/validate/validate.proto

```diff
@@ -559,10 +628,30 @@ message FloatRules {
   //   ];
   // }
   // ```
-  repeated float example = 9 [(priv.field).cel = {
+  repeated float example = 9 [(predefined).cel = {
     id: "float.example"
     expression: "true"
   }];
+
+  // Reserved for predefined rules using extension numbers defined within the
+  // [Protobuf Global Extension Registry][1]. Extension fields in this range
+  // that have the (buf.validate.predefined) option set will be treated as
+  // predefined field constraints that can then be set on field options of other
+  // messages to apply reusable field constraints.
+  //
+  // [1]: https://github.com/protocolbuffers/protobuf/blob/main/docs/options.md
+  extensions 1000 to 99999;
```

Comment:
I don't understand why we're splitting up the extension ranges into two sections - I've never seen that before. In the absence of a strong reason, coalesce.

Reply:
Was suggested [here](https://github.com/bufbuild/protovalidate/pull/246#discussion_r1747356550) and seemed reasonable enough to me. Either way, it's coalesced again now.